Gulpfile.js

```diff
@@ -161,4 +161,4 @@ gulp.task("default", [
   "clean:build",
   "watch",
   "server"
-]);
+]);
```

Comment:
Please be careful to not make extraneous changes to files that aren't part of your core check-in. Thank you!